ocaml/xapi/cpuid_helpers.ml

```diff
@@ -84,24 +85,25 @@ let is_strict_subset left right =
 
 (** Field definitions for checked string map access *)
 let features_t   = Map_check.pickler features_of_string string_of_features
-let features     = Map_check.field "features" features_t
-let features_pv  = Map_check.field "features_pv" features_t
-let features_hvm = Map_check.field "features_hvm" features_t
+let features = Map_check.field Xapi_globs.cpu_info_features_key features_t
```

Comment:
I think I prefer the older formatting given that most of the lines are the same. Helps to highlight the differences I think

Reply:
I've added some spaces to align as it was before